### DISCUSSION TITLE
GlobeViewport v8.4.0 State?

### DISCUSSION THREAD

**Discussion by jfuehner** [visgl/deck.gl#5424] [Q&A]
Hi All!

Saw with the release of [v8.4.0](https://github.com/visgl/deck.gl/releases/tag/v8.4.0) a new [GlobeView](https://deck.gl/examples/globe-view) example made it to the website. Super excited to see the globe progressing; great work!!

The [GlobeViewport ](https://deck.gl/docs/api-reference/core/globe-viewport) is still listed as _experimental_. I guess I am wondering at this point if it makes sense to start migrating from the [WebMercatorViewport](https://deck.gl/docs/api-reference/core/web-mercator-viewport) to the GlobeViewport.

Any insights into what is lacking from the GlobeViewport at this point? Just trying to get a better sense if I should wait before investing the effort to migrate... 

Thanks!

**Comment by Pessimistress** [comment]
https://deck.gl/docs/api-reference/core/globe-view#limitations
